hosting/src/components/Layouts/CmsLayout.tsx

```diff
@@ -5,7 +5,7 @@ import Header from "@/components/Header";
 import Sidebar from "@/components/Sidebar";
 import {useAuthentication} from "@/hooks/useAuthentication";
 import "flatpickr/dist/flatpickr.min.css";
-import "jsvectormap/dist/css/jsvectormap.css";
+// import "jsvectormap/dist/css/jsvectormap.css";
```

Comment:
We don't need this anymore?

Reply:
At the moment it doesn't look like we need it, I also don't know for sure when it will be used. I think I'll delete it and add it later if needed